recipes/libbasisu/all/conanfile.py

```diff
@@ -20,13 +20,15 @@ class LibBasisUniversalConan(ConanFile):
         "shared": [True, False],
         "use_sse4": [True, False],
         "with_zstd": [True, False],
+        "transcoder_only": [True, False],
         "custom_iterator_debug_level": [True, False]
     }
     default_options = {
         "fPIC": True,
         "shared": False,
-        "use_sse4": True,
+        "use_sse4": False,
         "with_zstd": True,
+        "transcoder_only": False,
```

Comment:
To match upstream project's actually behavior

```suggestion
        "transcoder_only": True,
```